debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -36,7 +36,7 @@
  */
 public class PostgresConnector extends SourceConnector {
 
-    private Logger logger = LoggerFactory.getLogger(getClass());
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
While at it, can you make it `private static final Logger LOGGER = ...`?

Reply:
yep. on it